src/System.Reflection/tests/FieldInfo/FieldInfo_ConstantTests.cs

```diff
@@ -68,11 +66,22 @@ public void TestSetValue_constbyteField()
             TestSetValue_constantField("byteField", (object)byte.MaxValue);
         }
 
-        //Verify SetValue method does not set value for RO Field
+        //Verify SetValue method _set_ set value for RO Field
```

Comment:
That first 'set' should have been 'does'.

The way the original delegated test (`TestSetValue_constant field`) was written, it masked that read-only fields weren't throwing the exceptions constant fields did.  (I ended up posting a short-lived issue because I had a bug in my repro code, but I digress)    I changed it to this form for documentation purposes.